kong/init.lua

```diff
@@ -660,6 +661,13 @@ function Kong.init()
   db:close()
 
   require("resty.kong.var").patch_metatable()
+
+  if config.privileged_agent and is_data_plane(config) then
+    local ok, err = process.enable_privileged_agent(2048)
```

Comment:
Is there any considerations that we set `2048` connections?

Reply:
@outsinre yes, the proper value should be figured out.

Reply:
How about adding a comments for it? such as `TODO: ...`

Reply:
Added.